tests/Schema/MySQLSchemaManagerTest.php

```diff
@@ -1,98 +0,0 @@
-<?php
-
-namespace Doctrine\DBAL\Tests\Schema;
-
-use Doctrine\Common\EventManager;
-use Doctrine\DBAL\Configuration;
-use Doctrine\DBAL\Connection;
-use Doctrine\DBAL\Driver;
-use Doctrine\DBAL\Platforms\AbstractMySQLPlatform;
-use Doctrine\DBAL\Schema\AbstractSchemaManager;
-use Doctrine\DBAL\Schema\ForeignKeyConstraint;
-use Doctrine\DBAL\Schema\MySQLSchemaManager;
-use PHPUnit\Framework\MockObject\MockObject;
-use PHPUnit\Framework\TestCase;
-
-use function array_map;
-
-class MySQLSchemaManagerTest extends TestCase
```

Comment:
This test covers the implementation details that have changed as of this PR. It was introduced in https://github.com/doctrine/dbal/pull/97 alongside an integration test that still passes.